apps/playground/src/main.css

```diff
@@ -0,0 +1,9 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+html {
+  font-family: system-ui, sans-serif;
+  color-scheme: dark light;
```

Comment:
does it have influence on our theming?

Reply:
It influences default colors that the browser provides (e.g. default background, text, scrolllbar, input, etc). I use it in every app I build.

I think we should add it to our theming because we are starting to use system colors in some places (e.g. I noticed `CanvasText` in the breadcrumbs PR).

Reply:
@FlyersPh9 You might want to check it then